trigger:
  off_int: 0
  on_int: 1
  trig_fail: 2

  button_short:
    button: 2

  button_long:
    button:
      id: 5
    
  button_full:
    button:
      id: 6
      event: "on_release"

  axis_range_half:
    axis_range:
      id: 4
      min: 8.0
      max: 2.0

  axis_range_invalid:
    axis_range:
      id: 5

  condition_simple:
    condition:
      conditions:
        - 1

  condition_recursive:
    condition:
      conditions:
        - 1
      operator: "nor"
      while_true:
        button: 0
      while_false:
        condition:
          conditions:
            - 1

  condition_invalid:
    condition:
      while_true: 1
      while_false: 0

value:
  constant_int: 0
  constant_float: 0.5
  constant_long:
    constant: 5.0

  axis_short:
    axis: 0

  axis_long:
    axis:
      id: 6

  axis_full:
    axis:
      id: 8
      min: 1
      max: -1

  condition_simple:
    condition:
      conditions:
        - 1

  condition_recursive:
    condition:
      conditions:
        - 1
      while_true:
        axis: 0
      while_false:
        condition:
          conditions:
            - 1

  condition_invalid:
    condition:
      while_true: 1
      while_false: 0

